_data/tool_and_resource_list.yml

```
@@ -2980,3 +3000,15 @@
   id: google-fit
   name: Google Fit
   url: https://www.google.com/fit/
+- description: MIxS - MIUViG (Minimum Information about an Uncultivated Virus Genome) is a checklist that is a part of the larger MIxS standard.
+  id: miuvig
+  name: Minimum Information about an Uncultivated Virus Genome (MIOViG)
+  registry:
+    fairsharing: bd9566
+  url: https://genomicsstandardsconsortium.github.io/mixs/0010012/
+- description: ORCID is an open, non-profit, community-driven registry of unique researcher identifiers and offers a transparent method of attributing outputs.
+  id: dublincore
+  name: Dublin Core Metadata Terms
+  registry:
+    fairsharing: 9vtwjs
+  url: hhttps://www.dublincore.org/
```

```suggestion
- description: To be replaced
  id: dublincore
  name: Dublin Core Metadata Terms
  registry:
    fairsharing: 9vtwjs
  url: hhttps://www.dublincore.org/
```

As you can see @bianchini88 This descitpion is for Dublin core, but it is a copy of ORCID

---

@bedroesb Sorry, I was seeing things that were not actually there. (I sometimes find it confusing that the first field we provide for a new item is its description and not the name or ID.)

Anyway, I have updated the description above.